tests/testAutopas/tests/tuning/tuningStrategy/PredictiveTuningTest.cpp

```diff
@@ -124,14 +134,20 @@ TEST_P(PredictiveTuningTest, testPredictions) {
 TEST_P(PredictiveTuningTest, testUnderAndOverflow) {
   auto extrapolationOption = GetParam();
   constexpr auto maxLong = std::numeric_limits<long>::max();
+  std::map<autopas::Configuration, long> expected = {
+      {_configurationLC_C01, 1}, {_configurationLC_C08, maxLong - 1}, {_configurationLC_Sliced, 101}};
+  // since lastResult is not actually an exprapolation, we expect the last values as prediction
+  if (extrapolationOption == autopas::ExtrapolationMethodOption::lastResult) {
+    expected = {{_configurationLC_C01, 10}, {_configurationLC_C08, maxLong - 100}, {_configurationLC_Sliced, 101}};
+  }
```

Comment:
```suggestion
  const std::map<autopas::Configuration, long> expected = [&](){
    // since lastResult is not actually an exprapolation, we expect the last values as prediction
    if (extrapolationOption == autopas::ExtrapolationMethodOption::lastResult) {
      return {{_configurationLC_C01, 10}, {_configurationLC_C08, maxLong - 100}, {_configurationLC_Sliced, 101}};
    } else {
      return {{_configurationLC_C01, 1}, {_configurationLC_C08, maxLong - 1}, {_configurationLC_Sliced, 101}};
    }
  }();
```
Wrap conditional initialization into a lambda so the variable can be `const`.
(Braces and formatting might be off...)

Reply:
done in [68558b4](https://github.com/AutoPas/AutoPas/pull/837/commits/68558b40210bf8ae8c1b486f0c1e315b6b1a9029)